jupyterhub/notebook-images/overlays/additional/elyra-notebook-imagestream.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  name: s2i-elyra-notebook
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - annotations:
+      openshift.io/imported-from: quay.io/akchinstc/s2i-elyra-notebook
```

Comment:
Can you please change the URL to `quay.io/thoth-station/s2i-lab-elyra:v0.0.1` (we'll update it once your other PR is merged)